### PR TITLE
Updated CVE-2019-14822's assignee and refs

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14822.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14822.json]: Invalid JSON file

**Comment by cristina479** [comment]
Red Hat CNA,

The JSON submitted doesn't conform with the current JSON schema. Can you make the corrections necessary in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

**Comment by cristina479** [comment]
Dear darunesh-RH,

Can you include the requested changes to the `version_data` and `description` fields?

**Comment by cristina479** [comment]
> Dear darunesh-RH,
> 
> Can you include the requested changes to the `version_data` and `description` fields?

This is a friendly reminder that we are still waiting on these changes.

**Comment by guitos** [comment]
@cristina479, the CVE-2019-14822 was updated via web from to include version information.

**Comment by guitos** [comment]
@cristina479, is there anything that needs to be updated for CVE-2019-14822? If not, can you please close this merge request?

File: 2019/14xxx/CVE-2019-14822.json (hunk 1 above)
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14822.json:4]: An exception of type JSONDecodeError occurred. Arguments:

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2019/14xxx/CVE-2019-14822.json (hunk 1 above)
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14822.json:4]: ("Expecting ',' delimiter: line 59 column 17 (char 1818)",)

Comment (fabioolive) [reply 2019/14xxx/CVE-2019-14822.json:4]: Fixed.

Comment (fabioolive) [reply 2019/14xxx/CVE-2019-14822.json:4]: Fixed.

**Comment by fabioolive** [comment]
@cve-team @cristina479 Hi, can we get a final review and merge of this branch? Thank you!

**Pull request closed without merging**